ansible/playbooks/aix.yml

```diff
@@ -136,12 +134,16 @@
       yum: name={{ item }} state=present update_cache=yes
       with_items:
         - http://www.oss4aix.org/download/RPMS/mktemp/mktemp-1.7-1.aix5.1.ppc.rpm
+        - http://www.bullfreeware.com/download/bin/2328/libiconv-1.14-22.aix6.1.ppc.rpm
         - http://www.bullfreeware.com/download/bin/2591/libunistring-0.9.6-2.aix6.1.ppc.rpm
         - http://www.bullfreeware.com/download/bin/3944/perl-5.24.0-3.aix6.1.ppc.rpm
         - http://www.oss4aix.org/download/RPMS/cmake/cmake-3.7.2-1.aix6.1.ppc.rpm
-      when: yum.stat.islnk is not defined
       tags: rpm_install
 
+    - name: Ensure /opt/freeware/bin is on the PATH
+      shell: mv /usr/bin/perl /usr/bin/perl.old && ln -s /opt/freeware/bin/perl /usr/bin/
```

Comment:
Why not just remove and recreate?  And you missed something on the end of the 'ln' cmd.

Reply:
In case you still want the older version, however the title (name) is miss leading I'll update that

`ln -s /opt/freeware/bin/perl /usr/bin/` works fine, you only need to add a file name at then end if you should to change the name.